listener.py

```
@@ -47,52 +47,67 @@ def build_client() -> tweepy.Client:
+        mention_id=mention.id,
+        author_id=mention.author_id,
+        conversation_id=mention.conversation_id,
+    )
+    member = route_mention(context)
+    print(
+        f"Mention {mention.id} routed to {member.profile.id} ({member.profile.kind})",
+        flush=True,
+    )
+    try:
+        reply = member.handle_mention(context)
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)